--- flake8.orig/setup.py   2017-08-08 02:31:43.000000000 +0200
+++ flake8/setup.py   2018-04-15 15:08:31.063294441 +0200
@@ -22,7 +22,7 @@
     # And in which releases we will update those ranges here:
     # http://flake8.pycqa.org/en/latest/internal/releases.html#releasing-flake8
     "pyflakes >= 1.5.0, < 1.7.0",
-    "pycodestyle >= 2.0.0, < 2.4.0",
+    "pycodestyle >= 2.0.0, < 2.5.0",
     "mccabe >= 0.6.0, < 0.7.0",
     "setuptools >= 30",
 ]
@@ -108,7 +108,8 @@
             PEP8_PLUGIN('module_imports_on_top_of_file'),
             PEP8_PLUGIN('compound_statements'),
             PEP8_PLUGIN('explicit_line_join'),
-            PEP8_PLUGIN('break_around_binary_operator'),
+            PEP8_PLUGIN('break_after_binary_operator'),
+            PEP8_PLUGIN('break_before_binary_operator'),
             PEP8_PLUGIN('comparison_to_singleton'),
             PEP8_PLUGIN('comparison_negative'),
             PEP8_PLUGIN('comparison_type'),
--- flake8.orig/setup.cfg	2017-10-23 11:16:02.000000000 +0000
+++ flake8/setup.cfg	2018-05-05 13:53:03.150753470 +0000
@@ -9,7 +9,7 @@
 	enum34; python_version<"3.4"
 	configparser; python_version<"3.2"
 	pyflakes >= 1.5.0, < 1.7.0
-	pycodestyle >= 2.0.0, < 2.4.0
+	pycodestyle >= 2.0.0, < 2.5.0
 	mccabe >= 0.6.0, < 0.7.0
 
 [egg_info]
